README.md

```diff
@@ -141,3 +141,14 @@ See IntegrationTestHelper class:
         return RedmineManagerFactory.createConnectionManagerWithExtraTrust(
                 Collections.singletonList(builtInExtension.get()));
     }
+
+## Using a custom (e.g. self-signed) SSL certificate with Client Certificate Authentication
+See IntegrationTestHelper class:
+
+    final Optional<KeyStore> builtInExtension = getExtensionKeystore();
```

Comment:
side note - I would skip "finals" here. those are not very useful in general (apart from class-level constant fields), and having them in docs adds cognitive load.

Reply:
Will do!